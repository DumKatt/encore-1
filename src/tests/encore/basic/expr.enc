class Main {
  def main () : void {
    print this.foo();
    print this.bar(this.foo());
    this.baz(0);
    this.baz(1)
  }

  def foo () : string {
    "Hello"
  }

  def bar (f:string) : int {
    print "Ponyworld!";
    2
  }

  def baz (b:int) : void {
    print (if b==0 then
             "Hello"
           else
             "Ponyworld!")
  }
}
