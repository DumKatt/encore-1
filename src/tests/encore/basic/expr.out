Hello
Ponyworld!
2
Hello
Ponyworld!
